sysutils/dnscontrol/Portfile

```
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/StackExchange/dnscontrol 3.17.0 v
+
+checksums           rmd160  d3db682a9b741c54c3c07d1c72499e85a3d3286d \
+                    sha256  6a0b73973db57b22e50dd7350b3889dc7d9054a8205d0bd6fcf4650aaeb7f36b \
+                    size    1997040
+
+homepage            https://stackexchange.github.io/dnscontrol/
+description         Synchronize your DNS to multiple providers from a simple DSL
+long_description    DNSControl is a system for maintaining DNS zones. It has   \
+                    two parts: a domain specific language (DSL) for describing \
+                    DNS zones plus software that processes the DSL and pushes  \
+                    the resulting zones to DNS providers such as Route53,      \
+                    Cloudflare, and Gandi. It can send the same DNS records to \
+                    multiple providers. It even generates the most beautiful   \
+                    BIND zone files ever. It runs anywhere Go runs (Linux,     \
+                    macOS, Windows). The provider model is extensible, so more \
+                    providers can be added.
+
+categories          sysutils
+license             MIT
+maintainers         @ajhall \
```

I'm new to MacPorts, so please let me know the etiquette on maintainership. I'm OK with maintaining this port on a best-effort volunteer basis, but it's an active open source project with many contributors (https://github.com/StackExchange/dnscontrol) and I don't want to step on any toes. Is it OK to volunteer myself here? (cc: @tlimoncelli)

---

It's perfectly fine to volunteer as maintainer for this port, especially since you're submitting it.

In general though, the entries for each maintainer in the `maintainer` block are surrounded with curly braces:

```suggestion
maintainers         {@ajhall} \
```

---

> Is it OK to volunteer myself here? (cc: @tlimoncelli)

I'm ok with it!  I appreciate any help I can get!

---

Thanks, happy to help!